src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -560,9 +560,9 @@
 
 helps['ad sp delete'] = """
 type: command
-short-summary: Delete a service principal and its role assignments.
+short-summary: Delete a service principal.
 examples:
-  - name: Delete a service principal and its role assignments. (autogenerated)
+  - name: Delete a service principal. (autogenerated)
```

Comment:
We can delete `(autogenerated)` and L567 once this command has been reviewed by human.

```suggestion
  - name: Delete a service principal.
```